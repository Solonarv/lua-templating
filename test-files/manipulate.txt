<%
-- check the `with_buffer` inspection method
-- also checks that start/end tags work properly inside functions
%>this text should be reversed<%
with_buffer(function (str) return string.reverse(str) end)
%>
more normal text here....
blah blah. now we should see a bracketed copy of everything so far:
<%
with_buffer(function (str)
    emit("[[\n")
    emit(str)
    emit("\n]]")
end) %>
more end text, after the brackets
