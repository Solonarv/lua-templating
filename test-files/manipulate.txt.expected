desrever eb dluohs txet siht
more normal text here....
blah blah. now we should see a bracketed copy of everything so far:
[[
desrever eb dluohs txet siht
more normal text here....
blah blah. now we should see a bracketed copy of everything so far:

]]
more end text, after the brackets
